app/mailers/mailshots_mailer.rb

```diff
@@ -9,4 +9,18 @@ def community_launch
     subject = "Forum, Swag, Automated Feedback, and a new Dig Deeper section"
     mail_to_user(@user, subject)
   end
+
+  def company_support_donor
+    @user = params[:user]
+
+    subject = "Could your company support Exercism?"
+    mail_to_user(@user, subject, reply_to: "loretta@exercism.org")
```

Comment:
I've set the `reply_to` address to loretta's email. Is that okay?

Reply:
Yes. thanks.